### ISSUE TITLE
When PageBuilder is active, I only get blank page

### ISSUE THREAD

**Issue by JohnnyB826** [ThemeFuse/Unyson#3655]
When PageBuilder is active, I can't edit a page or create a new one - all I get is a blank screen! 
I'm running latest version of Wordpress, Unyson, and all plugins are up to date. I've cleared cache as well. 

Please help!!!!!!

 I'd be happy to pass along login credentials.

**Comment by streivie** [comment]
I have the same problem. Have you solved this somehow already?

**Comment by JohnnyB826** [comment]
No. Never got a response either. Ended up switching to another theme altogether. 

 

Johnny Burnham

Marketing & Community Development Director

Northwest CT YMCA
259 Prospect St., Torrington, CT 06790
Office (860) 489-3133 x 132

Fax: (860) 482-4853

 <http://www.nwcty.org> nwctY.org | Facebook | Twitter 

y_logo_hl_email_sig      af00_r_blu_rgb.jpg

**Comment by streivie** [comment]
Ah ok. Thanks anyway. I'll probably do the same.
